htdocs/main.php

```diff
@@ -288,6 +288,15 @@ function tplFromRequest($param)
 
 $tpl_data['css'] = $config->getSetting('css');
 
+// Prints SQL Queries if a file with output exists
+if (isset($_SESSION['file_sql_queries'])) {
```

Comment:
@jacobpenny Could you mark this as 'Changes Requested' if you feel this change is significant? Otherwise, could you approve as a reviewer?